test/stancjs/stancjs.expected

```diff
@@ -246,29 +246,35 @@ $ node pedantic.js
 ["Warning in 'string', line 4, column 9: The variable tt may not have been\n    assigned a value before its use."]
 []
 $ node removed.js
-Syntax error in 'string', line 3, column 7 to column 8, parsing error:
+Syntax error in 'string', line 3, column 8 to column 10, parsing error:
    -------------------------------------------------
      1:  
      2:  parameters {
      3:  	real y[10];
-                ^
+                 ^
      4:  }
```

Comment:
note: these look off because a tab counts as one space when positioning the caret. Presumably this has always been an issue. Tabs are rarely used so maybe not worth fixing.